addons/point_of_sale/static/src/xml/Popups/ClosePosPopup.xml

```diff
@@ -75,17 +75,21 @@
                                 </tbody>
                             </t>
                             <tbody t-if="otherPaymentMethods.length &gt; 0">
-                                <tr t-foreach="otherPaymentMethods" t-as="pm" t-key="pm.name">
+                                <tr t-foreach="otherPaymentMethods" t-as="pm" t-key="pm.id">
                                     <td t-esc="pm.name"/>
                                     <td t-esc="env.pos.format_currency(pm.amount)"/>
+                                    <t t-set="_showDiff" t-value="_getShowDiff(pm)" />
+                                    <td t-if="_showDiff" t-on-input="handleInputChange(pm.id)"><input class="pos-input" type="number" t-model.number="state.payments[pm.id].counted"/></td>
+                                    <td t-if="_showDiff" t-esc="env.pos.format_currency(state.payments[pm.id].difference)" t-att-class="{'warning': state.payments[pm.id].difference}"/>
                                 </tr>
                             </tbody>
                         </table>
                     </div>
                     <textarea placeholder="Notes" class="closing-notes" t-model="state.notes"/>
-                    <div t-if="cashControl" class="accept-closing" t-att-class="{invisible: !state[defaultCashDetails.name].difference}">
-                        <input type="checkbox" id="accept" t-model="state.acceptClosing"/>
-                        <label for="accept">Accept <t t-esc="defaultCashDetails.name"/> difference and post a profit/loss journal entry</label>
+                    <div class="accept-closing" t-att-class="{invisible: !hasDifference()}">
+                        <input t-att-disabled="!hasUserAuthority()" type="checkbox" id="accept" t-model="state.acceptClosing"/>
+                        <label t-att-class="{disabled: !hasUserAuthority()}" for="accept">Accept payments difference and post a profit/loss journal entry</label>
+                        <div class="warning" t-att-class="{invisible: hasUserAuthority()}">The maximum difference allowed is <t t-esc="amountAuthorizedDiff"/>. You can contact your manager to force it.</div>
```

Comment:
I just saw what you did with the `_showDiff` above, do you mind doing the same here? This avoid calling the same function thrice.